### PR TITLE
Fix for issue 54

### PR DESCRIPTION
**Pull request by Mikepicker**
- nickpass has now a default value (the tutorial in the readme fails since one has to specify a value for nickpass)
- readme updated with this new change

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -63,7 +63,7 @@ Congratulations, you just wrote an IRC bot!
 
 #### Legobot.legobot()
 
-`Legobot.legobot(host,port,nick,chans, logfunc = "", hostpw = "", defaultFunc = None, defaultFuncChar = "")`
+`Legobot.legobot(host,port,nick,chans, nickpass = "", logfunc = "", hostpw = "", defaultFunc = None, defaultFuncChar = "")`
```
Comment (bbriggs) [inline README.md]: We just merged this change in another PR. If we don't have any conflicts, I'll try to leave this part be, but odds are we'll have to drop this part.

File: build/lib/Legobot/legoCron.py [hunk 2]
```diff
@@ -0,0 +1,158 @@
+from datetime import datetime, timedelta
```
Comment (bbriggs) [inline build/lib/Legobot/legoCron.py]: Same. Need to drop this file.

File: build/lib/Legobot/__init__.py [hunk 3]
```diff
@@ -0,0 +1,313 @@
+import socket
```
Comment (bbriggs) [inline build/lib/Legobot/__init__.py]: It looks like you rolled this with pip and that generated a `build/lib/...` directory. 

Can you drop these stuff in there from the commit and then add `build/lib/*` to gitignore in a separate PR?

Comment (Mikepicker) [inline build/lib/Legobot/__init__.py]: I opened a new PR [here](https://github.com/bbriggs/Legobot/pull/73) :)

**Comment by bbriggs** [comment]
See #73

**Pull request closed without merging**